### PR TITLE
Bug 1973491: jsonnet: update deps

### PR DESCRIPTION
**Pull request by prashbnair**
* [ ] I added CHANGELOG entry for this change.
* [ ] No user facing changes, so no entry in CHANGELOG was needed.

### CODE REVIEW

File: jsonnet/jsonnetfile.lock.json [hunk 1]
```diff
@@ -142,8 +142,8 @@
           "subdir": "jsonnet/prometheus-operator"
         }
       },
-      "version": "2388bfa557c9836d5ff01e620a129d33852670ff",
-      "sum": "eHJp7oFWvBEsSmwoRML356DLK80n7rRt8XKRZ+YawvQ="
+      "version": "3f047bc9c74105ba774aff726798a66b81409f7f",
+      "sum": "TcXPVhn0BIFCf5mWa6hyIljVl13ItwPjyeiwro56kjE="
```
Comment (simonpasquier) [inline jsonnet/jsonnetfile.lock.json:146]: this is pulling new CRD versions that aren't supported by the version of prometheus-operator currently shipped

Comment (prashbnair) [reply jsonnet/jsonnetfile.lock.json]: Reverted this change.

File: assets/node-exporter/prometheus-rule.yaml [hunk 2]
```diff
@@ -226,6 +226,30 @@ spec:
...
+        description: File descriptors limit at {{ $labels.instance }} is currently
+          at {{ printf "%.2f" $value }}%.
+        summary: Kernel is predicted to exhaust file descriptors limit soon.
+      expr: |
+        (
+          node_filefd_allocated{job="node-exporter"} * 100 / node_filefd_maximum{job="node-exporter"} > 90
+        )
+      for: 15m
+      labels:
+        severity: critical
```
Comment (simonpasquier) [inline assets/node-exporter/prometheus-rule.yaml:252]: cc @raptorsun this is a new alert to take care of :)

**Pull request merged**